paperqa/agents/tools.py

```diff
@@ -399,6 +400,253 @@ async def complete(
         return f"{'Success' if has_successful_answer else 'Unsure'} | {state.status}"
 
 
+class ClinicalTrialsSearch(NamedTool):
+    TOOL_FN_NAME = "clinical_trials_search"
+
+    model_config = ConfigDict(extra="forbid", arbitrary_types_allowed=True)
+
+    search_count: int = 8
+    previous_searches: dict[str, int] = Field(default_factory=dict)
+    settings: Settings = Field(default_factory=Settings)
+
+    # Gather evidence tool must be modified to understand the new evidence
+    GATHER_EVIDENCE_TOOL_PROMPT_OVERRIDE: ClassVar[str] = (
+        """
+        Gather evidence from previous papers and clinical trials given a specific question.
+
+        Will increase evidence, relevant paper counts, and relevant clinical trial counts.
+        A valuable time to invoke this tool is right after another tool increases paper or clinical trials count.
+        Feel free to invoke this tool in parallel with other tools, but do not call this tool in parallel with itself.
+        Only invoke this tool when the paper count or clinical trial count is above zero, or this tool will be useless.
+
+        Args:
+            question: Specific question to gather evidence for.
+            state: Current state.
+
+        Returns:
+            String describing gathered evidence and the current status.
+        """
+    )
+
+    async def clinical_trials_search(self, query: str, state: EnvironmentState):
```

Comment:
```suggestion
    async def clinical_trials_search(self, query: str, state: EnvironmentState) -> str:
```

Even though we don't use return type now, it's still nice to have for IDE usage